hijack/views.py

```diff
@@ -46,3 +27,23 @@ def release_hijack(request):
 def disable_hijack_warning(request):
     request.session['display_hijack_warning'] = False
     return redirect_to_next(request, default_url='/')
+
+
+def login_with_id(request, user_id):
+    warnings.warn('"login_with_id" has been deprecated in favor of "acquire_user_view".', DeprecationWarning)
+    return acquire_user_view(request, user_id)
```

Comment:
acquire_user_view() takes 1 positional argument but 2 were given

Reply:
it seems such an error we should have noticed at the test level

Reply:
Agreed, I will write more tests.

Reply:
@codingjoe Please ping me when this bug will be solved.

Reply:
@Mogost resolved. I also expanded the tests to catch the error.

Reply:
Thank you. I will retest this branch in a week.